examples/todos/src/containers/VisibleTodoList.spec.js

```diff
@@ -0,0 +1,37 @@
+import { getVisibleTodos } from './VisibleTodoList'
+import { VisibilityFilters } from '../actions'
+
+let todos = [{id: 1, completed: false, text: 'a test text'}]
+
+describe('get visible todo list', () => {
+  it('should filter SHOW_ALL', () => {
+    expect(
+        getVisibleTodos(todos, VisibilityFilters.SHOW_ALL)
+    ).toEqual(todos)
+  })
+
+  it('should filter SHOW_COMPLETED', () => {
+    expect(
+        getVisibleTodos(todos, VisibilityFilters.SHOW_COMPLETED)
+    ).toEqual([])
+  })
+
+  it('should filter SHOW_ACTIVE', () => {
+    expect(
+        getVisibleTodos(todos, VisibilityFilters.SHOW_ACTIVE)
+    ).toEqual(todos)
+  })
+
+  it('should throw Exception', () => {
+    let filter = 'NOT A VALID FILTER'
+
+    expect(() => {
+        getVisibleTodos(todos, filter)
```

Comment:
Why we need to nested inside a function instead of
```
expect(getVisibleTodos(todos, filter)). toThrow(...)
```

Reply:
I've just follow this example:
https://jestjs.io/docs/en/expect#tothrowerror

And it won't work unless I put a nested function inside. 
Do you know why?

Reply:
Ah oki, you're right, it only work when we put a nested function inside. So please keep your code 😄